src/deploy/hosting/convertConfig.ts

```diff
@@ -57,27 +60,99 @@ export async function convertConfig(
     return out;
   }
 
-  const endpointBeingDeployed = (serviceId: string, region: string = "us-central1") => {
+  const endpointFromBackend = (
+    targetBackend: backend.Backend,
+    serviceId: string,
+    platform?: string,
+    region?: string
+  ): backend.Endpoint | undefined => {
```

Comment:
Since you've got a couple places where `platform` isn't specified, but `region` is, why not make this take an option array so you can more easily call the function (and stop writing `undefined` in a  few places)

```suggestion
  const endpointFromBackend = (opts: {
    targetBackend: backend.Backend;
    serviceId: string;
    platform?: string;
    region?: string;
  }): backend.Endpoint | undefined => {
```

Reply:
Created a new interface for this because it's used in multiple places.